projects/storefrontlib/src/lib/cms/store/selectors/component.selectors.ts

```diff
@@ -11,20 +11,17 @@ export const getComponentState: MemoizedSelector<
   (state: fromFeature.CmsState) => state.component
 );
 
-export const getComponentEntities: MemoizedSelector<any, any> = createSelector(
-  getComponentState,
-  fromComponent.getComponentEntities
-);
+export const getComponentEntities: MemoizedSelector<
+  any,
+  { [id: string]: any }
```

Comment:
`any` can be Generic type T.

Reply:
a Generic type T of what? Component?
Also (assuming you mean <T extends Component>) then I do not know the proper syntax to implement it (I tried a few, none worked)